packages/fxa-settings/src/pages/ConnectAnotherDevice/en.ftl

```diff
@@ -0,0 +1,25 @@
+## Connect Another Device page
+# A user will only see this header if they are signed in. The header will be preceded by a green checkmark (rtl/ltr sensitive)
+connect-another-device-signed-in-header = <subheader>You’re signed into Firefox<subheader>
+# A "success" message visible to users who verified via email
+connect-another-device-email-confirmed-banner = Email confirmed
+# A "success" message visible to users who verified via sign-in
+connect-another-device-signin-confirmed-banner = Sign-in confirmed
+# A message prompts the user to sign in to this instance of the Firefox browser so as to complete device sync. This is followed by a link labeled "Sign in"
+connect-another-device-signin-to-complete-message = Sign in to this Firefox to complete set-up
+# A link for the user to sign in to the current Firefox browser, preceded by a message prompting the user to sign in so as to complete the device sync setup
+connect-another-device-signin-link = Sign in
+# A message prompting the user to sign in via a different device than the current one so as to complete the device-syncing process
+connect-another-device-still-adding-devices-message = Still adding devices? Sign in to Firefox on another device to complete set-up
```

Comment:
nit: `set-up` is the British variant. Should this be 'setup'?

Reply:
Ohhhh interesting! I just copy and pasted this but I can fix it!

Reply:
Yikes!